alacritty_terminal/src/event_loop.rs

```diff
@@ -339,13 +340,37 @@ impl event::Notify for Notifier {
             return;
         }
 
-        self.0.send(Msg::Input(bytes));
+        let _ = self.0.send(Msg::Input(bytes));
     }
 }
 
 impl event::OnResize for Notifier {
     fn on_resize(&mut self, window_size: WindowSize) {
-        self.0.send(Msg::Resize(window_size));
+        let _ = self.0.send(Msg::Resize(window_size));
+    }
+}
+
+#[derive(Debug)]
+pub enum EventLoopSendError {
+    Io(io::Error),
+    Send(mpsc::SendError<Msg>),
+}
+
+impl Display for EventLoopSendError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            EventLoopSendError::Io(x) => x.fmt(f),
+            EventLoopSendError::Send(x) => x.fmt(f),
+        }
+    }
+}
+
+impl std::error::Error for EventLoopSendError {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        match self {
+            EventLoopSendError::Io(x) => Some(x),
+            EventLoopSendError::Send(x) => Some(x),
```

Comment:
`x` is not a good variable name.

Reply:
Renamed to `err` to conform with other `std::error::Error` `impl`s in the project.